app/services/access_token_verifier.rb

```diff
@@ -12,16 +12,16 @@ def initialize(http_authorization_header)
   end
 
   def submit
```

Comment:
bonus points for documenting! maybe this is too verbose, but I feel like tuple responses are a surprise so better to document a little if we can
```suggestion
  # @return [Array(FormResponse, ServiceProviderIdentity), Array(FormResponse, nil)]
  def submit
```